### ISSUE TITLE
List of languages without a colour

### ISSUE THREAD

**Issue by wopian** [github/linguist#5003]
Continuing from #4506. 49% (280) languages currently do not have a language assigned and show as grey (a lot of these are grouped languages that GitHub shows ungrouped in search results).

List generated with https://gist.github.com/wopian/8639e5d24119cbd99548b73c18321686

## Early Sidenotes

- Languages without a logo or no other clear branding are being skipped. 

- If the language has no logo and is developed by a sole company, their logo/brand is used.

- If a logo consists of multiple colours, I've listed them by most dominant (or in the case of Blitz, least blinding).

- Will try and do 2 or more letters per day.

## Key

🙅 = No logo, no brand or lack of info for language
❓ = No suitable colour, but is grouped into a language that has a colour

### Numbers

- [ ] 4D `#004289` (source: [brand](https://4d.com))

### A

- [ ] ~ABNF~ 🙅
- [ ] ~ASL~ 🙅
- [ ] ~ASN.1~ 🙅 
- [ ] Adobe Font Metrics `#fa0f00` (source: Adobe)
- [ ] Alpine Abuild `#0D597F` (grouped in `Shell`, source: [Alpine logo](#0D597F))
- [ ] Altium Designer `#A89663` (source: [Altium Designer logo](https://en.wikipedia.org/wiki/Altium_Designer))
- [ ] Ant Build System `#A9157E` (source: [Apache Ant logo](https://en.wikipedia.org/wiki/Apache_Ant))
- [ ] ApacheConf `#D12127` (source: [Apache logo](http://httpd.apache.org/docs/2.4/configuring.html)
- [x] ~Apollo Guidance Computer (grouped in `Assembly`)~ #5001
- [ ] ~AsciiDoc~ 🙅 Blue header? https://asciidoc.org/
- [ ] Augeas `#9CC134`, `#65a703` or `#ebf8d5` (source: [logo and brand](https://augeas.net/docs/language.html))
- [ ] Avro IDL `#0040FF` (source: [Apache Avro logo](https://avro.apache.org/docs/1.9.2/idl.html))
- [ ] ~Awk~ 🙅

### B

- [ ] ~Befunge~ 🙅
- [ ] BibTeX (grouped in `TeX`) ❓ Use green from `Tex`? (logo is black text)
- [x] ~Bison (grouped in `Yacc`) ❓ Use colour from `Yacc`? (logo is a black & white image)~ #5001
- [ ] ~BitBake~ 🙅
- [ ] BlitzBasic `#00D8FF`, `#00FFAE` (source: [Blitz BASIC logo](https://en.wikipedia.org/wiki/Blitz_BASIC) - jeez)
- [ ] Bluespec `#12223C` (source: [Bluespec logo](https://github.com/bluespec))
- [ ] Brightscript `#662D91` (source: [Roko Brightscript logo](https://developer.roku.com/en-gb/docs/references/brightscript/language/brightscript-language-reference.md))

### C

- [ ] C-ObjDump ❓ Use colour from `C`?
- [ ] C2hs Haskell (grouped in `Haskell`) ❓ Use purple from `Haskell`?
- [ ] ~CLIPS~ 🙅 Maybe http://www.clipsrules.net/ https://sourceforge.net/projects/clipsrules/
- [ ] CMake `#A3A3A3`, `#01BD00`, `#3132B9` or `#DA3434` (source: [logo](https://en.wikipedia.org/wiki/CMake))
- [ ] ~COBOL~ 🙅
- [ ] COLLADA `#F1A42B` (source: [logo](https://en.wikipedia.org/wiki/COLLADA))
- [ ] ~CSV~ 🙅
- [ ] ~CWeb~ 🙅
- [ ] Cabal Config `#483465` (source: [brand colours](https://www.haskell.org/cabal/))
- [ ] Cap'n Proto `#C42727` (source: [brand colours](https://capnproto.org/))
- [ ] CartoCSS ❓ Use colour from `CSS`?
- [ ] ~Charity~ 🙅
- [ ] ChucK `#3F8000`, `#80FF00` or `#FF8500` (source: [logo](https://en.wikipedia.org/wiki/ChucK))
- [ ] Closure Templates (grouped in `HTML`) ❓ Use colour from `Closure` or `HTML`?
- [ ] Cloud Firestore Security Rules `#FFCA28`, `#FFA000` or `#F57C00` (source: [Firebase branding](https://firebase.google.com/brand-guidelines))
- [ ] ~CoNLL-U~ 🙅
- [ ] CodeQL `#5C31FF`, `#5c31ff` or `#151046` (source: https://semmle.com/codeql)
- [x] ~ColdFusion CFC (grouped in `ColdFusion`)~ #5001
- [ ] ~Cool~ 🙅
- [ ] Coq `#CFB486` or `#CCA073` (source: [logo](https://en.wikipedia.org/wiki/Coq))
- [ ] Cpp-ObjDump ❓ Use colour from `C++`?
- [ ] ~Creole~ 🙅
- [ ] Csound `#343433` (source: [logo](https://csound.com/))
- [ ] Csound Document `#343433` (source: [logo](https://csound.com/))
- [ ] Csound Score `#343433` (source: [logo](https://csound.com/))
- [ ] ~Cycript~ 🙅
- [ ] Cython `#3572A5` (grouped in `Python`)
- [ ] ~cURL Config (grouped in `INI`)~ 🙅

### D

- [ ] D-ObjDump ❓ Use colour from `D`?
- [ ] ~DIGITAL Command Language~ 🙅
- [ ] ~DNS Zone~ 🙅
- [ ] ~DTrace~ 🙅 Appears to have a mascot that uses an orange/pink/purple theme
- [ ] Darcs Patch (source: [logo](https://en.wikipedia.org/wiki/Darcs), but its a gradient from white to 100% green)
- [ ] ~Diff~ 🙅
- [ ] DirectX 3D File (source: [logo](https://en.wikipedia.org/wiki/DirectX), another white to green gradient)
- [ ] ~desktop~ 🙅
- [ ] ~dircolors~ 🙅

### E

- [ ] ~EBNF~ 🙅
- [ ] ECLiPSe `#AFEEFF`, `#00FFFF` or `#0057D7` (grouped in `prolog`, source: [logo and brand](http://eclipseclp.org/))
- [x] ~EJS (grouped in `HTML`)~ #5001
- [ ] ~EML~ 🙅
- [ ] ~Eagle~ 🙅
- [ ] Easybuild `#2A72ED`, `#069406` or `#B50811` (grouped in `Python`, source: [logo](https://easybuild.readthedocs.io/en/latest/))
- [ ] Ecere Projects `#913960` (grouped in `JavaScript`) Configuration for `eC` (use same as language itself)
- [ ] EditorConfig `#E0EFEF`, `#FEFEFE` or `#FFF2F2` (grouped in `INI`, source: [logo and brand](https://editorconfig.org/))
- [ ] ~Edje Data Collection~ 🙅
- [ ] ~edn~ 🙅

### F

- [ ] FIGlet Font `#FFDDBB` (source: [brand](http://www.figlet.org/))
- [ ] Filebench WML `#FFC000` (source: [org logo](https://github.com/filebench))
- [ ] ~Filterscript (grouped in `RenderScript`)~ 🙅
- [ ] ~Formatted~ 🙅
- [ ] Fortran Free Form `#4d41b1` (grouped in `Fortran`, source: same as `Fortran`)
- [ ] fish `#4AAE47` or `#4A9BC8` (grouped in `Shell`, source: [logo](https://fishshell.com/))

### G

- [ ] GAMS `#F39619` or `#2b1a02` (source: [logo](https://www.gams.com/img/gams_logo.svg))
- [ ] GAP `#0000ff` or `#ccffff` (source: [site](https://www.gap-system.org))
- [ ] GCC Machine Description `#FFCFAB`, `#AB9A7E` or `#F39105` (source: [logo](https://en.wikipedia.org/wiki/GNU_Compiler_Collection))
- [ ] ~GDB~ 🙅 Black & white logo
- [ ] GEDCOM `#003058` OR `#86B940`, `#9C947A` or `#BBB7B1` (first source: [logo of LDS Church (developers)](https://en.wikipedia.org/wiki/The_Church_of_Jesus_Christ_of_Latter-day_Saints), second source: [logo of FamilySearch (operated by LDS and uses GEDCOM today)](https://en.wikipedia.org/wiki/GEDCOM#Versions)
- [ ] GLSL `#5586A4` (source: [openGL brand](https://www.khronos.org/legal/trademarks/))
- [ ] ~GN~ 🙅 Generate Ninja - No logo or branding
- [ ] Genshi `#cc0000` or `#D5200C` (source: [branding and Edgewall logo](https://genshi.edgewall.org/))
- [ ] Gentoo Ebuild `#beb8db`, `#54487a` or `#9400FF` (grouped in `Shell`, source: [brand](https://www.gentoo.org/))
- [ ] Gentoo Eclass `#beb8db`, `#54487a` or `#9400FF` (grouped in `Shell`, source: [brand](https://www.gentoo.org/))
- [ ] Gerber Image `#D01F25` or #D20B00` (source: [logo or branding](https://www.ucamco.com/en/gerber))
- [ ] Gettext Catalog `#a42e2b` (source: [GNU branding](https://www.gnu.org/software/gettext/manual/index.html))
- [ ] Git Attributes `#F44D27` (grouped in `INI`, source: [logo](https://git-scm.com/))
- [ ] Git Config `#F44D27` (grouped in `INI`, source: [logo](https://git-scm.com/))
- [ ] Glyph Bitmap Distribution Format `#fa0f00` (source: Adobe)
- [ ] Grace `#615f8b` (source: [brand](http://gracelang.org/applications/))
- [ ] Gradle `#02303A` (source: [logo and brand](https://gradle.org/))
- [ ] ~Graph Modeling Language~ 🙅
- [ ] GraphQL `#e10098` (source: [logo](https://graphql.org/img/logo.svg))
- [ ] Graphviz (DOT) ❓ [complex logo](https://graphviz.org/) with conflicting site branding (and a different favicon logo)
- [ ] Groovy Server Pages (grouped in `Groovy`) `#76A8B8` (source: [repo logo](https://github.com/apache/groovy)) OR `#4298B8` or `#46A5C8` (source: [branding and site logo](https://groovy-lang.org/))
  - Groovy itself is orange on Linguist for some reason and I don't think it ever had an orange logo or branding going on - so may want to opt for Groovy's orange or raise an issue with Groovy maintainers/users

### H

- [ ] HAProxy `#3D6DA5` (source: [centre square from logo](https://www.haproxy.org/))
- [ ] HCL `#000000` (source: [logo and brand](https://www.hashicorp.com/))
- [ ] ~HLSL~ 🙅 No logo, proprietary shader language for DirectX by Microsoft
- [ ] HTML+Django `#20AA76` or `#0C4B33` (grouped in `HTML`, source: [branding](https://www.djangoproject.com/))
- [ ] HTML+ECR `#000100` or `#2E1052` (grouped in `HTML`, source: [logo (matching Crystal on Linguist) or brand accent](https://crystal-lang.org/api/0.35.1/ECR.html))
- [ ] HTML+EEX `#6e4a7e` (grouped in `HTML`, source: [logo (matching Elixir on Linguist)](https://elixirschool.com/en/lessons/specifics/eex/))
- [ ] HTML+ERB `#701516` (grouped in `HTML`, source: matching Ruby on Linguist)
- [ ] HTML+PHP `#4F5D95` (grouped in `HTML`, source: matching PHP on Linguist)
- [ ] HTML+Razor `#512BD4` (grouped in `HTML`, source: [.NET logo](https://github.com/dotnet) and [branding](https://dotnet.microsoft.com/))
- [ ] ~HTTP~ 🙅
- [ ] HXML `#f68712` or `#DF7B12` (source: [Haxe logo](https://haxe.org/manual/compiler-usage-hxml.html))
- [ ] ~HyPhy~ 🙅

### I

- [ ] ~INI~ 🙅
- [ ] ~IRC log~ 🙅
- [ ] Ignore List `#F44D27` (grouped in `INI`) (source: [logo](https://git-scm.com/)) `.gitignore`
- [ ] Inform 7 ❓ [complex logo](http://inform7.com/)
- [ ] Inno Setup `#264b99` (source: [brand](https://jrsoftware.org/isinfo.php))
- [ ] Isabelle ROOT `#FEFE00`, `#CECEE7` or `#CE4A4A` (grouped in `Isabelle`, source: yellow matching Isabelle on Linguist, [other two from logo](https://isabelle.in.tum.de/))

### J

- [x] ~JFlex (grouped in `Lex`)~ #5001
- [ ] JSON `#3e3e3e` (source: average of gradient logo](https://www.json.org/json-en.html))
- [ ] JSON with Comments `#3e3e3e` (grouped in `JSON`) ❓ Same as JSON
- [ ] JSON5 `#267CB9` (source: [branding](https://json5.org/))
- [ ] JSONLD `#0C479C` (source: [logo](https://github.com/json-ld)
- [ ] JSX `#f1e05a` (grouped in `JavaScript`, source: matching Javascript on Linguist)
- [ ] Jasmin ❓ [complex logo](http://jasmin.sourceforge.net/)
- [ ] Java Properties `#b07219` (source: matching Java on Linguist)
- [ ] Java Server Pages `#b07219` OR `#f98200` or `#FEB940` (grouped in `Java`, source: matching Java on Linguist OR [Jakarta logo or branding](https://jakarta.ee/))
  - Java Server Pages appears to have been rebranded[(https://en.wikipedia.org/wiki/Jakarta_Server_Pages) as Jakarta Server Pages at some point
- [ ] JavaScript+ERB `#701516` (grouped in `JavaScript`, source: matching Ruby on Linguist)
- [ ] Jison `#56B3CB` (grouped in `Yacc`, source: [logo and branding](https://zaa.ch/jison/docs/)) 
- [ ] Jison Lex `#56B3CB` (grouped in `Lex`, source: [logo and branding](https://zaa.ch/jison/docs/)) 

### K

- [ ] KiCad Layout `#2f4aaa`, `#1a2a60` or `#f77300` (source: [logo](https://gitlab.com/kicad))
- [ ] KiCad Legacy Layout `#2f4aaa`, `#1a2a60` or `#f77300` (source: [logo](https://gitlab.com/kicad))
- [ ] KiCad Schematic `#2f4aaa`, `#1a2a60` or `#f77300` (source: [logo](https://gitlab.com/kicad))
- [ ] Kit `#2deada`, `#1dba9d` or `#227d6b` (source: [logo](https://www.kitlang.org/))

### L

- [ ] ~LTspice Symbol~ 🙅
- [ ] LabVIEW `#fede06` (source: [logo](https://en.wikipedia.org/wiki/LabVIEW#/media/File:LabVIEW_Logo.jpg))
- [ ] ~Lean~ 🙅
- [ ] LilyPond `#587E55` (source: average of [logo](https://lilypond.org/))
- [ ] ~Limbo~ 🙅
- [ ] ~Linker Script~ 🙅
- [ ] ~Linux Kernel Module~ 🙅
- [ ] Liquid `#3399cc` (source: [branding](https://shopify.github.io/liquid/))
- [ ] Literate Agda `#315665`) (grouped in `Agda`, source: same as Agda in Linguist)
- [ ] Literate CoffeeScript `#244776` (grouped in `CoffeeScript`, source: same as CoffeeScript in Linguist)
- [ ] Literate Haskell `#5e5086` (grouped in `Haskell`, source: same as Haskell in Linguist)
- [ ] ~Logos~ 🙅
- [ ] ~Logtalk~ 🙅 [solid black logo](https://github.com/LogtalkDotOrg)
- [ ] ~LoomScript~ 🙅 [solid black logo](http://loomsdk.com/)

### M

- [ ] M
- [ ] M4
- [ ] M4Sugar (grouped in `M4`)
- [ ] MUF (grouped in `Forth`)
- [ ] Mako
- [ ] Mathematica
- [ ] Maven POM
- [ ] MediaWiki
- [ ] Microsoft Developer Studio Project
- [ ] MiniD
- [ ] Modelica
- [ ] Modula-2
- [ ] Module Management System
- [ ] Monkey
- [ ] Moocode
- [ ] MoonScript
- [ ] Motorola 68K Assembly (grouped in `Assembly`)
- [ ] Muse
- [ ] Mustache (grouped in `HTML`)
- [ ] Myghty
- [ ] mupad

### N

- [ ] NASL
- [ ] NEON
- [ ] NL
- [ ] NPM Config (grouped in `INI`)
- [ ] NSIS
- [ ] Nginx
- [ ] Ninja
- [x] ~NumPy (grouped in `Python`)~ #5001
- [ ] nanorc (grouped in `INI`)

### O

- [ ] ObjDump
- [ ] Object Data Instance Notation
- [ ] Opa
- [ ] Open Policy Agent
- [ ] OpenCL (grouped in `C`)
- [ ] OpenEdge ABL
- [ ] OpenRC runscript (grouped in `Shell`)
- [ ] OpenSCAD
- [ ] OpenStep Property List
- [ ] OpenType Feature File
- [ ] Org
- [ ] Ox

### P

- [ ] PLpgSQL
- [ ] POV-Ray SDL
- [ ] Parrot Assembly (grouped in `Parrot`)
- [ ] Parrot Internal Representation (grouped in `Parrot`)
- [ ] Pic (grouped in `Roff`)
- [ ] Pickle
- [ ] PicoLisp
- [ ] PlantUML
- [ ] Pod
- [ ] Pod 6
- [ ] Pony
- [ ] PostCSS (grouped in `CSS`)
- [ ] Proguard
- [ ] Protocol Buffer
- [ ] Public Key
- [ ] Pure Data
- [ ] Python console (grouped in `Python`)
- [ ] Python traceback (grouped in `Python`)

### Q

- [ ] QMake

### R

- [ ] RDoc
- [ ] REALbasic
- [ ] REXX
- [ ] RMarkdown
- [ ] RPC
- [ ] RPM Spec
- [ ] Raw token data
- [ ] Readline Config (grouped in `INI`)
- [ ] Redcode
- [ ] Regular Expression
- [ ] ~RenderScript~ 🙅
- [ ] Rich Text Format
- [ ] RobotFramework
- [ ] Roff Manpage (grouped in `Roff`)
- [ ] reStructuredText

### S

- [ ] SMT
- [ ] SPARQL
- [ ] SQL
- [ ] SQLPL
- [ ] SSH Config (grouped in `INI`)
- [ ] STON (grouped in `Smalltalk`)
- [ ] SWIG
- [ ] Sage
- [ ] Scaml (grouped in `HTML`)
- [ ] Scilab
- [ ] ShaderLab
- [ ] ShellSession
- [ ] Sieve
- [ ] Smali
- [ ] Smarty
- [ ] Spline Font Database
- [ ] Stata
- [ ] SubRip Text
- [ ] SugarSS (grouped in `CSS`)

### T

- [ ] TLA
- [ ] TOML
- [ ] TSQL
- [ ] TSV
- [ ] TSX (grouped in `TypeScript`)
- [ ] TXL
- [ ] Tcsh (grouped in `Shell`)
- [ ] Tea
- [ ] Texinfo
- [ ] Text
- [ ] Textile
- [ ] Thrift
- [ ] Turtle
- [ ] Type Language

### U

- [x] ~Unified Parallel C (grouped in `C`)~ #5001
- [ ] Unity3D Asset
- [ ] Unix Assembly (grouped in `Assembly`)
- [ ] UrWeb

### V

- [ ] Vim Help File
- [ ] Vim Snippet

### W

- [ ] Wavefront Material
- [ ] Wavefront Object
- [ ] Web Ontology Language
- [ ] WebIDL
- [ ] WebVTT
- [ ] Wget Config (grouped in `INI`)
- [ ] Windows Registry Entries
- [ ] World of Warcraft Addon Data

### X

- [ ] X BitMap (grouped in `C`)
- [ ] X Font Directory Index
- [ ] X PixMap (grouped in `C`)
- [ ] XCompose
- [ ] XML
- [ ] XML Property List (grouped in `XML`)
- [ ] XPages
- [ ] XProc
- [ ] XS
- [ ] Xojo
- [ ] Xtend

### Y

- [ ] YANG

### Z
- [ ] Zeek
- [ ] Zimpl

**Comment by wopian** [comment]
Added C to D

**Comment by smola** [comment]
Gentoo Ebuild/Eclass could be`#9400FF` (source: [gentoo.org](https://www.gentoo.org/). AFAIK that's the primary color, while there's a secondary green.

**Comment by wopian** [comment]
Added E, F and Gentoo Ebuild/Eclass

**Comment by Haroenv** [comment]
- [ ] GAMS `#F39619` or `#2b1a02` (source: [logo](https://www.gams.com/img/gams_logo.svg))
- [ ] GAP `#0000ff` or `#ccffff` (source: [site](https://www.gap-system.org), probably too bright)
- [ ] GCC Machine Description ?
- [ ] GDB ?
- [ ] GEDCOM ?
- [ ] GLSL `#5586A4` (source: [openGL brand](https://www.khronos.org/legal/trademarks/))
- [ ] GN ? maybe ninja logo?
- [ ] Genshi `#b00` (source: [site](https://genshi.edgewall.org), although the red isn't in the logo)
- [ ] Gentoo Ebuild `#beb8db`, `#54487a` or `#9400FF` (grouped in Shell, source: [brand](https://www.gentoo.org/))
- [ ] Gentoo Eclass `#beb8db`, `#54487a` or `#9400FF` (grouped in Shell, source: [brand](https://www.gentoo.org/))
- [ ] Gerber Image `#D20B00` (source: [site](https://www.ucamco.com/en/gerber))
- [ ] Gettext Catalog ?
- [ ] Git Attributes (grouped in INI)
- [ ] Git Config (grouped in INI)
- [ ] Glyph Bitmap Distribution Format `#000` (source: no colors in [spec](https://www.adobe.com/content/dam/acom/en/devnet/font/pdfs/5005.BDF_Spec.pdf))
- [ ] Grace `#615f8b` (source: [blog](http://gracelang.org/applications/))
- [ ] Gradle `#02303A` (source: [site](https://gradle.org))
- [ ] Graph Modeling Language ?
- [ ] GraphQL `#e10098` (source: [logo](https://graphql.org/img/logo.svg))
- [ ] Graphviz (DOT) ?
- [ ] Groovy Server Pages (grouped in Groovy) `#4298b8` (source: [groovy site](https://groovy-lang.org))

**Comment by wopian** [comment]
Added G, H, I and J. Incorporating suggestions from @Haroenv

**Comment by Alhadis** [comment]
> GN ? maybe ninja logo?

What logo? Neither Ninja nor GN have logos of their own. I made [this one](https://github.com/file-icons/icons/blob/76bd3c4f8b5be540b47771b0d53e6a435425c4c2/svg/GN.svg) several years back for the `file-icons` project, but it doesn't have a colour. Though I'd say deep, dark blue would be the appropriate colour for evoking midnight-draped *shinobi*. 😉


> a lot of these are grouped languages that GitHub shows ungrouped in search results

IMHO, grouped languages should inherit the colour of their parent language. They're represented on GitHub as their parent, so it makes sense they should inherit their parent's colour.

> Glyph Bitmap Distribution Format `#000` (source: no colours in spec)

Adobe invented that format, so we can just use `#fa0f00` (taken from [Adobe's logo](https://www.adobe.com/content/dam/cc/icons/Adobe_Corporate_Horizontal_Red_HEX.svg))


> Git Attributes `#F44D27` (grouped in INI, source: logo)
> Git Config `#F44D27` (grouped in INI, source: logo)

🤔 INI? Wait, that doesn't sound right... they might be config formats, but calling them INI is a stretch…

**Comment by wopian** [comment]
> > Glyph Bitmap Distribution Format `#000` (source: no colours in spec)
> 
> Adobe invented that format, so we can just use `#fa0f00` (taken from [Adobe's logo](https://www.adobe.com/content/dam/cc/icons/Adobe_Corporate_Horizontal_Red_HEX.svg))

I used the PostScript colour on Linguist as Adobe Postscript's logo was on both (that and Adobe Font Metrics) specification PDFs in greyscale. 

I presumed they were the same shade of red but I guess it turns out Adobe's logo is brighter than Adobe PostScript's Linguist colour - likely a result of the axed colour similarity check 👀 

> > Git Attributes `#F44D27` (grouped in INI, source: logo)
> > Git Config `#F44D27` (grouped in INI, source: logo)
> 
> 🤔 INI? Wait, that doesn't sound right... they might be config formats, but calling them INI is a stretch…

The grouped data is extracted from `languages.yml`, but yeah I too was surprised they were grouped under INI when I got to them.

**Comment by Alhadis** [comment]
> I presumed they were the same shade of red but I guess it turns out Adobe's logo is brighter than Adobe PostScript's Linguist colour

Yup. Back in the day, Adobe's branding used a darker shade of red that matched PostScript's. They brightened that at some point in the last 30 years, but since PostScript's no longer an actively supported product, it still uses the old branding colour.

In any case, PostScript and GBDF have very, very little in common, so it's probably better to use a different colour anyway. 😁

> but yeah I too was surprised they were grouped under INI when I got to them.

This might warrant a closer look. Some entries, like EditorConfig and NPM configs, are grouped under INI and only exist for dedicated highlighting purposes. But we can't go slapping the INI label anything that happens to be an unindented config file...

**Comment by wopian** [comment]
Added K and L and changed the Adobe languages to use Adobe's branding instead of Adobe PostScript.

>@Alhadis: This might warrant a closer look. Some entries, like EditorConfig and NPM configs, are grouped under INI and only exist for dedicated highlighting purposes. But we can't go slapping the INI label anything that happens to be an unindented config file...

Should I open another issue to get discussion going for this? Since the group associations are in the Linguist YML file and out of scope for this issue.

>>@wopian: a lot of these are grouped languages that GitHub shows ungrouped in search results

>@Alhadis: IMHO, grouped languages should inherit the colour of their parent language. They're represented on GitHub as their parent, so it makes sense they should inherit their parent's colour.

I'm somewhat conflicted on this but more skewed to only using the parent if there's no suitable colour/branding for the sub-language. Django, Embedded Crystal/Elixir/Ruby, PHP and Razor are my only concerns.

- 👍 Quite a few grouped languages have clear branding which makes sense when viewed from the repository languages section (e.g [fish](https://github.com/fish-shell/fish-shell/search?l=fish))
  - 👍Several grouped under INI have clear branding, but INI itself does not
- 👍 For HTML and the embedded programming languages I'm personally more interested in the embedded language than the mark-up language itself.
  - 👎 However, an embedded programming language inside another programming language like JavaScript with Embedded Ruby doesn't gel well with that method. It'd stick with the JavaScript colour here...which leads to
  - 👎 Most of the embedded language template file is going to be standard HTML (or JavaScript for `JavaScript+Ruby`), with maybe only a few lines of template tag syntax

**Comment by Nixinova** [comment]
This will need to be updated once #5476 is merged

**Comment by Nixinova** [comment]
<details>
<summary>
Updated list of languages without colours
</summary>

- 4D
- ABNF
- ASL
- ASN.1
- Adobe Font Metrics
- Alpine Abuild
- Altium Designer
- Ant Build System
- ApacheConf
- AsciiDoc
- Augeas
- Avro IDL
- Befunge
- BibTeX
- BitBake
- BlitzBasic
- Bluespec
- Brightscript
- C-ObjDump
- C2hs Haskell
- CIL
- CLIPS
- CMake
- COBOL
- CODEOWNERS
- COLLADA
- CWeb
- Cabal Config
- Cap'n Proto
- CartoCSS
- Charity
- ChucK
- Cloud Firestore Security Rules
- CoNLL-U
- CodeQL
- Cool
- Coq
- Cpp-ObjDump
- Creole
- Csound
- Csound Document
- Csound Score
- Cue Sheet
- Cycript
- Cython
- D-ObjDump
- DIGITAL Command Language
- DNS Zone
- DTrace
- Darcs Patch
- Diff
- DirectX 3D File
- E-mail
- EBNF
- ECLiPSe
- Eagle
- Easybuild
- Ecere Projects
- Edje Data Collection
- FIGlet Font
- Filebench WML
- Filterscript
- Formatted
- Fortran Free Form
- GAMS
- GAP
- GCC Machine Description
- GDB
- GEDCOM
- GN
- Genshi
- Gerber Image
- Gettext Catalog
- Glyph Bitmap Distribution Format
- Graph Modeling Language
- HAProxy
- HCL
- HTTP
- HyPhy
- IRC log
- Inform 7
- Isabelle ROOT
- Kit
- Kusto
- LTspice Symbol
- Lean
- Limbo
- Linker Script
- Linux Kernel Module
- Logos
- Logtalk
- LoomScript
- M
- M4
- M4Sugar
- MUF
- Mako
- Mathematica
- Maven POM
- Microsoft Developer Studio Project
- Microsoft Visual Studio Solution
- MiniD
- Modelica
- Modula-2
- Module Management System
- Monkey
- Moocode
- MoonScript
- Motorola 68K Assembly
- Muse
- Myghty
- NASL
- NEON
- NL
- NSIS
- Nginx
- Ninja
- ObjDump
- Object Data Instance Notation
- Opa
- Open Policy Agent
- OpenCL
- OpenEdge ABL
- OpenRC runscript
- OpenSCAD
- OpenStep Property List
- OpenType Feature File
- Ox
- POV-Ray SDL
- Parrot Assembly
- Parrot Internal Representation
- Pic
- Pickle
- PicoLisp
- PlantUML
- Pod
- Pod 6
- Pony
- Proguard
- Protocol Buffer
- Public Key
- Pure Data
- QMake
- REALbasic
- REXX
- RPC
- RPM Spec
- Raw token data
- Readline Config
- Redcode
- Redirect Rules
- RenderScript
- Rich Text Format
- RobotFramework
- SELinux Policy
- SMT
- SPARQL
- SSH Config
- STON
- SWIG
- Sage
- Scilab
- ShellSession
- Sieve
- Smali
- Smarty
- Soong
- Spline Font Database
- Stata
- TLA
- TXL
- Tcsh
- Tea
- Texinfo
- Text
- Textile
- Thrift
- Turtle
- Type Language
- Unity3D Asset
- Unix Assembly
- UrWeb
- Vim Help File
- Vim Snippet
- Wavefront Material
- Wavefront Object
- Web Ontology Language
- WebIDL
- WebVTT
- Wget Config
- Windows Registry Entries
- World of Warcraft Addon Data
- X BitMap
- X Font Directory Index
- X PixMap
- XCompose
- XPages
- XProc
- XS
- Xojo
- Xtend
- YANG
- Zeek
- Zimpl
- cURL Config
- desktop
- dircolors
- edn
- fish
- mupad
- robots.txt
</details>

**Comment by Nixinova** [comment]
Okay I've now looked through every item in the list and attached a relevant colour #5476, this should be done.

**Comment by Nixinova** [comment]
<details><summary>
Current list of languages without colours
</summary>
- ABNF
- ASL
- ASN.1
- Befunge
- C-ObjDump
- C2hs Haskell
- CIL
- COBOL
- CODEOWNERS
- CartoCSS
- Charity
- CoNLL-U
- Cool
- Cpp-ObjDump
- Creole
- Cue Sheet
- Cycript
- Cython
- D-ObjDump
- DIGITAL Command Language
- DNS Zone
- DTrace
- Diff
- E-mail
- EBNF
- Eagle
- Edje Data Collection
- Filterscript
- Formatted
- GDB
- GN
- Gettext Catalog
- Glyph Bitmap Distribution Format
- Graph Modeling Language
- HCL
- HTTP
- HyPhy
- IRC log
- Inform 7
- Kit
- Kusto
- LTspice Symbol
- Lean
- Limbo
- Linker Script
- Linux Kernel Module
- Logos
- LoomScript
- M
- M4
- M4Sugar
- MUF
- Maven POM
- Microsoft Developer Studio Project
- Microsoft Visual Studio Solution
- MiniD
- Module Management System
- Monkey
- Moocode
- Muse
- Myghty
- NASL
- NEON
- NL
- NSIS
- Ninja
- ObjDump
- Object Data Instance Notation
- Opa
- OpenRC runscript
- OpenStep Property List
- OpenType Feature File
- Ox
- Parrot Assembly
- Parrot Internal Representation
- Pic
- Pickle
- PlantUML
- Pod
- Pod 6
- Pony
- Proguard
- Protocol Buffer
- Public Key
- Pure Data
- QMake
- REALbasic
- RPC
- RPM Spec
- Raw token data
- Readline Config
- Redcode
- Redirect Rules
- RenderScript
- Rich Text Format
- SELinux Policy
- SMT
- SSH Config
- STON
- SWIG
- Sage
- ShellSession
- Sieve
- Smali
- Soong
- Spline Font Database
- Tcsh
- Tea
- Texinfo
- Text
- Turtle
- Type Language
- Unix Assembly
- Wavefront Material
- Wavefront Object
- WebIDL
- WebVTT
- Wget Config
- X BitMap
- X Font Directory Index
- X PixMap
- XCompose
- XPages
- XProc
- XS
- YANG
- Zeek
- cURL Config
- desktop
- dircolors
- edn
- robots.txt
</details>

**Comment by wopian** [comment]
Thank you for finishing this issue up for me @Nixinova 🎉

**Comment by mxmilkiib** [comment]
Turtle is still lacking a colour. Example projects; https://github.com/moddevices/mod-lv2-data https://github.com/davemollen/dm-Octaver
